tests/testsuite.src/syn_occurs.at

```diff
@@ -501,3 +501,107 @@ AT_CHECK([$COMPILE_ONLY -std=mf -fodoslide prog.cob ], [0], [], [])
 
 AT_CLEANUP
 
+
+AT_SETUP([SET to out-of-range literal])
+AT_KEYWORDS([SET])
+
+AT_DATA([prog.cob], [
+       IDENTIFICATION   DIVISION.
+       PROGRAM-ID.      prog.
+       DATA             DIVISION.
+       WORKING-STORAGE  SECTION.
+       01 G.
+         02 X           PIC X OCCURS 10 INDEXED I.
+       PROCEDURE        DIVISION.
+           SET I TO ZERO.
+           SET I TO 11.
+           STOP RUN.
+])
+
+AT_CHECK([$COMPILE_ONLY prog.cob], [0], [],
+[prog.cob:9: warning: SET I TO 0 is out of bounds
+prog.cob:10: warning: SET I TO 11 is out of bounds
+])
+
+AT_CLEANUP
+
+AT_SETUP([SET index])
+AT_KEYWORDS([SET])
```

Comment:
please use a double empty line before the new test (as above) and comment out the unused keyword (possibly also in the test case above "ODO")

Reply:
> comment out the unused keyword (possibly also in the test case above "ODO")

I'm not sure to get this; could you be more specific ?

Reply:
`AT_KEYWORDS([SET])` is unused because the word SET is already in the "list of words used in AT_SETUP", which are automatically included as keywords. If you don't have another keyword you would want to run this test with just comment the complete AT_KEYWORDS line